src/components/quiz-question/quiz-question-wrapper.tsx

```diff
@@ -134,7 +134,7 @@ export const QuizQuestionWrapper = withText(translates)((props: QuizQuestionWrap
       case KalturaQuizQuestionTypes.MultiChoice:
         return <MultiChoice {...questionProps} />;
       case KalturaQuizQuestionTypes.Reflection:
-        return <ReflectionPoint {...questionProps} />;
+        return <ReflectionPoint {...questionProps} continueButton={continueButtonRef}/>;
```

Comment:
instead of pass `continueButtonRef` inside ReflectionPoint component I would recommend handle focus inside QuizQuestionWrapper. You can add extra check for question type inside next condition: 
```
      if (qui.a) {
        continueButtonRef.current?.focus();
      }
```

Reply:
done